taskfiles/utils.yml

```diff
@@ -5,20 +5,19 @@ tasks:
   # CHECKSUM UTILS
   # ===
 
-  # @param {string} DATA_DIR The directory to compute the checksum for.
+  # @param {string[]} DATA_PATHS List of paths to compute the checksum for.
   # @param {string} OUTPUT_FILE
-  # @param {[]string} [EXCLUDE_PATHS] A list of paths, relative to `DATA_DIR`, to exclude from the
-  # checksum.
+  # @param {string[]} [EXCLUDE_PATHS] List of paths, relative to any `DATA_PATHS`, to exclude from
+  # the checksum.
   compute-checksum:
-    desc: "Tries to compute a checksum for the given directory and output it to a file."
+    desc: "Tries to compute a checksum for the given paths and output it to a file."
     internal: true
     silent: true
     requires:
-      vars: ["DATA_DIR", "OUTPUT_FILE"]
+      vars: ["DATA_PATHS", "OUTPUT_FILE"]
     cmds:
       - >-
         tar cf -
```

Comment:
Should we add `--wildcards` and `--no-anchored` like you do for the `download-and-extract-tar` task?